plot-api/src/main/kotlin/jetbrains/letsPlot/scale/SizeContinuous.kt

```diff
@@ -28,8 +28,9 @@ import jetbrains.letsPlot.intern.Scale
  *
  * @param expand A numeric vector of length two giving multiplicative and additive expansion constants.
  * @param naValue Missing values will be replaced with this value.
- * @param guide TBD
- * @param trans TBD
+ * @param guide A function used to create a guide (guide_colorbar(), guide_legend()) or its name ("colorbar", "legend").
+ *      Defaults to "none".
```

Comment:
The default is not "none". I think we don't need to mention default value at all in this doc.
But we have to mention that "none" will hide the guide.